solidity/package.json

```diff
@@ -28,6 +28,7 @@
   "dependencies": {
     "@keep-network/tbtc": ">1.1.2-dev <1.1.2-ropsten",
     "@openzeppelin/contracts": "^4.1.0",
+    "@summa-tx/bitcoin-spv-sol": "https://gitpkg.now.sh/keep-network/bitcoin-spv/solidity?solidity/v3.1.0-solidity-0.8",
```

Comment:
This is such an important component that we should use a more structured, official path and make sure we are ready to add more updates in the future if needed.

My suggestion would be:
- Branch off in `keep-network/bitcoin-spv` from `v3.1.0` tag with `releases/mainnet/solidity/v3.1.0-solc-0.8` branch.
- Branch off with a feature branch from `releases/mainnet/solidity/v3.1.0-solc-0.8`.
- Cherry-pick changes from https://github.com/keep-network/bitcoin-spv/compare/solidity/v3.1.0...keep-network:solidity/v3.1.0-solidity-0.8.
- Open official PR from the feature branch to `releases/mainnet/solidity/v3.1.0-solc-0.8`.
- Go through the review process and have the PR approved and merged.
- Update version in `package.json` to `v3.1.0-solc-0.8` on the release branch.
- Tag `solidity/v3.1.0-solc-0.8` on `releases/mainnet/solidity/v3.1.0-solc-0.8` release branch.
- Push `@keep-network/bitcoin-spv-sol` in version `v3.1.0-solc-0.8` to NPM.
- Refer to `"@keep-network/bitcoin-spv-sol": "3.1.0-solc-0.8",` here.
- Open an issue in `summa-tx/bitcoin-spv-sol` and suggest creating a release branch from `v3.1.0`, and that we can open there a PR with our changes to enable solc 0.8 support.

Reply:
Yep, totally right but can we do all that work in a follow-up PR? This one is already big and starts to become a bag of loosely related changes.

Reply:
Done in https://github.com/keep-network/tbtc-v2/pull/90/commits/c0dcb790b63d7ac2ae89ec3fa593ae66fb95b3ad and updated the PR description.